client/internal/engine_test.go

```diff
@@ -196,6 +197,7 @@ func TestEngine_UpdateNetworkMap(t *testing.T) {
 		WgPort:       33100,
 	}, nbstatus.NewRecorder())
 	engine.wgInterface, err = iface.NewWGIFace("utun102", "100.64.0.1/24", iface.DefaultMTU)
+	engine.routeManager = routemanager.NewManager(ctx, key.PublicKey().String(), engine.wgInterface, engine.statusRecorder)
```

Comment:
Some tests test route management in isolation from the whole system, but none test it working with the Engine. 
Route management is functionality that is just like the SSH server shouldn't disturb connectivity and updates. 
It is crucial to test because Engine is responsible for triggering route updates when an update comes from the management.